exonum-java-binding/core/rust/src/runtime/cmd.rs

```diff
@@ -42,36 +40,6 @@ const JVM_ARGS_APPEND: &str = "JVM_ARGS_APPEND";
 // TOML keys for EJB configuration
 pub const EJB_CONFIG_SECTION_NAME: &str = "ejb";
 
-/// Encapsulates processing of extensions of the `finalize` command. At this step we gather some of
```

Comment:
Docs at the start of the file also need changes

Reply:
Done.